web/src/components/AccountSetting/AccountSetting.js

```diff
@@ -0,0 +1,60 @@
+import { useForm } from 'react-hook-form'
+import { Form, Label, TextField, FieldError, Submit } from '@redwoodjs/forms'
+import { useMutation } from '@redwoodjs/web'
+import { useAuth } from '@redwoodjs/auth'
+import { toast, Toaster } from '@redwoodjs/web/toast'
+
+const UPDATE_ACCOUNT = gql`
+  mutation UpdateAccountMutation($input: UpdateUserInput!) {
+    updateUser(input: $input) {
+      name
+    }
+  }
+`
+const AccountSetting = () => {
```

Comment:
Can we rename `AccountSetting` to `AccountSettings` everywhere? This will require some renaming of files too!